librosa/core/audio.py

```diff
@@ -496,6 +496,8 @@ def resample(
                 - ``res_type='sinc_best'``, ``'sinc_medium'``, or ``'sinc_fastest'``: for high-, medium-,
                   and low-quality sinc interpolation
 
+            To use `soxr`, set ``res_type='soxr_hq'`` or ``res_type='soxr_vhq'`` (fast)
```

Comment:
Are these all the supported modes?

Reply:
i would like to see a link where these HQ and VHQ modes are described.
besides, are you sure that `soxr_vhq` is the faster option? that seems backward

Reply:
python-soxr supports QQ, LQ, MQ, HQ, VHQ.
I concerned about description can be too verbose.

`soxr_hq` is faster than `soxr_vhq`. I meant that both soxr options are fast.
Speed benchmark in order of QQ, LQ, MQ, HQ, VHQ
```
100 loops, best of 5: 4.9 ms per loop
100 loops, best of 5: 8.32 ms per loop
100 loops, best of 5: 8.77 ms per loop
100 loops, best of 5: 9.28 ms per loop
10 loops, best of 5: 23.6 ms per loop
```

Reply:
I think we could keep it relatively short, but still list the supported modes.  Something like: 
```
res_type='soxr_vhq', 'soxr_hq', 'soxr_mq', 'soxr_lq', 'soxr_qq' for very high quality, high quality, medium quality, low quality, or q??? quality
```

Reply:
Refined docstring of `resample()`.